src/lib.rs

```diff
@@ -113,6 +118,10 @@ pub trait Actionlike:
 ///
 /// Use with [`InputManagerPlugin`](crate::plugin::InputManagerPlugin), providing the same enum type to both.
 #[derive(Bundle)]
+#[deprecated(
+    since = "0.17.0",
```

Comment:
Given the nature of the breaking change, I have added the next major version here